backend/src/apis/KlicktippController.ts

```diff
@@ -75,9 +99,25 @@ export const tagUser = async (email: string, tagIds: string): Promise<boolean> =
 }
 
 export const getKlicktippTagMap = async () => {
+  if (!CONFIG.KLICKTIPP) {
+    return ''
+  }
   const isLogin = await loginKlicktippUser()
   if (isLogin) {
     return await klicktippConnector.tagIndex()
   }
   return ''
 }
+
+export const addFieldsToSubscriber = async (
+  email: string,
+  fields: any = {},
+  newemail = '',
+  newsmsnumber = '',
+) => {
+  const isLogin = await loginKlicktippUser()
```

Comment:
This is true in case of no klicktipp active, therefore the following code is called

Reply:
Yeah I corrected this!